src/ReadModel/ServiceProvider.php

```diff
@@ -17,12 +19,19 @@ class ServiceProvider extends \Illuminate\Support\ServiceProvider
     public function register()
     {
         $this->app->singleton(MethodNameInflector::class, ProjectClassNameInflector::class);
+
+        $this->app->singleton('read_model.repository', function () {
```

Comment:
This shouldn't be bound to the `RepositoryInterface` as it returns the configuration for the driver. Perhaps this shouldn't bind anything yet and the application should use it in their own service provider?